field/src/extension/cubic.rs

```diff
@@ -5,26 +5,26 @@ use core::ops::{Add, AddAssign, Div, DivAssign, Mul, MulAssign, Neg, Sub, SubAss
 use rand::distributions::Standard;
 use rand::prelude::Distribution;
 
-use crate::extension::OptimallyExtendable;
+use crate::extension::BinomiallyExtendable;
 use crate::field::Field;
 use crate::{AbstractExtensionField, AbstractField};
 
 #[derive(Copy, Clone, Eq, PartialEq, Hash)]
-pub struct CubicOef<F: OptimallyExtendable<3>>(pub [F; 3]);
+pub struct CubicBEF<F: BinomiallyExtendable<3>>(pub [F; 3]);
```

Comment:
nit: we've been using the convention of camel case acronyms (like `CubicBef`) based on these [guidelines](https://rust-lang.github.io/api-guidelines/naming.html#casing-conforms-to-rfc-430-c-case)

Reply:
Thx, applied the camel case convention